docs/codesnippets/615_import_firefox_session_sqlite.py

```diff
@@ -1,9 +1,15 @@
+#!/usr/local/bin/python3
```

Comment:
Could we remove the shebang here? The snippet is included verbatim in the documentation, and this adds two lines without adding any real value, I think.